### PR TITLE
feat: add typography & lists

### PR DESCRIPTION
**Pull request by anik31**
In components folder typography.css & list.css contains the css for typography & lists respectively.
docs.html contains the html & documentation for all the components.

Deploy preview for typography- https://deploy-preview-7--vacay-ui.netlify.app/docs#Typography
Deploy preview for lists - https://deploy-preview-7--vacay-ui.netlify.app/docs#List

# Typography images 
image
image

# Lists images 
image
image

### CODE REVIEW

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    font-weight: 700;
+}
+
+.list-notification>li>ul>li:last-child{
+    color: var(--text-gray);
+}
+
+
+.list-notification>li>ul>li{
+    margin: 0.3rem;
```
Comment (b0llu) [inline components/list.css:41]: you can use px if the size is less than 1rem

File: components/typography.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    line-height: 2rem;
+}
+
+h4, .text-sm{
+    font-size: 1rem;
+    line-height: 1.5rem;
+}
+
+h5,.text-xsm{
+    font-size: 0.5rem;
```
Comment (b0llu) [inline components/typography.css:22]: here too you can change this to px

File: components/vacay.css [hunk 3]
```diff
@@ -48,7 +50,7 @@
     --red-color-400:#EF4444;
     --red-color-900:#7F1D1D;
     --maroon-color:#800000;
-
+    --text-gray:#3E5060;
     --border-radius: 0.5rem;
```
Comment (b0llu) [inline components/vacay.css:54]: rem to px. Also use variable for color

File: style.css [hunk 4]
```diff
@@ -154,7 +154,7 @@ button{
 }
 
 .sidenav li{
-    padding: 1rem;
+    padding: 0.8rem;
```
Comment (b0llu) [inline style.css:157]: rem to px

File: style.css [hunk 5]
```diff
@@ -191,8 +191,4 @@ iframe{
 
 .example-desc{
     margin: 0.5rem 0;
```
Comment (b0llu) [inline style.css:193]: rem to px

File: components/list.css (hunk 1 above)
Comment (anik31) [reply components/list.css:41]: Thanks for the suggestion dhruv but I prefer using rems as it is relative.

**Review by 05him — APPROVED** [review]
your code looks good just one  suggestion 
image
that here you can use (xl,l,m,s,xs) instead of (xlg,lg,md,sm,xms) as these are already specified form their names ,

**Comment by anik31** [comment]
> your code looks good just one suggestion image that here you can use (xl,l,m,s,xs) instead of (xlg,lg,md,sm,xms) as these are already specified form their names ,

@05him I will prefer using the existing ones as they are more understandable & I have also seen some css libraries who follow this naming.

**Review by mikesch803 — APPROVED** [review]
you can merge, your code looks good

**Pull request merged**